src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,48 @@
+spring:
```

Comment:
This file shouldn't be present, this would be generated when deployment takes place

Reply:
removed